test/rijksbot/rijksbot_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+require 'rijksbot'
+
+describe Rijksbot, :vcr do
+
+  let(:twitter_client){ Minitest::Mock.new } # I mocked the TwitterClient as I do not know and do not want to expose the bot's twitter credentials and I cannot signup for a new Twitter APP at this time. We might want to try to catch the Twitter requests on VCR as well. cbuggle. 03.05.2018
```

Comment:
mocking is exactly the right way, you don't want your tests communicate to third-parties

Reply:
What I actually wanted to do is record the Twitter communication  once using VCR and use an actual genuine response.